tests/dag_processing/test_processor.py

```diff
@@ -124,6 +124,13 @@ def test_dag_file_processor_sla_miss_callback(self, mock_get_dagbag, create_dumm
 
         session.merge(TaskInstance(task=task, execution_date=test_start_date, state="success"))
         session.merge(SlaMiss(task_id="dummy", dag_id="test_sla_miss", execution_date=test_start_date))
+        session.commit()
+
+        slas_miss_objects: list[SlaMiss] = (
+            session.query(SlaMiss)
+            .filter(SlaMiss.notification_sent == False, SlaMiss.dag_id == dag.dag_id)  # noqa
+            .all()
+        )
```

Comment:
Why are we doing this? I feel we should only touch notifiers

Reply:
Was done to make sure the arguments are handled correctly, but now that we are not touching `dag_processor` this is no longer needed.

Reply:
Removed this change